### PR TITLE
Added Pytorch for deep learning in machine-learning section

### PR DESCRIPTION
**Pull request by chinmaydas96**
Added Deep learning using PyTorch.

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/english/machine-learning/pytorch-for-deep-learning/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Deep Learning using PyTorch
+---
+
+
+#### What is PyTorch?
+"PyTorch is open source deep learning platform primarily developed by Facebook's artificial-intelligence research group,that provides a seamless path from research prototyping to production deployment which has been adopted by many (if not most) of the researchers that we most respect."
```
Comment (rupali317) [inline guide/english/machine-learning/pytorch-for-deep-learning/index.md]: Can we rephrase this as:
"PyTorch is **an** open source deep learning platform **, which is** primarily developed by Facebook's artificial intelligence research group **. It** provides......."

File: guide/english/machine-learning/pytorch-for-deep-learning/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Deep Learning using PyTorch
+---
+
+
+#### What is PyTorch?
+"PyTorch is open source deep learning platform primarily developed by Facebook's artificial-intelligence research group,that provides a seamless path from research prototyping to production deployment which has been adopted by many (if not most) of the researchers that we most respect."
+
+
+* In PyTorch things are way more imperative and dynamic: you can define, change and execute nodes as you go, no special session interfaces or placeholders like in Tensorflow. Overall, the framework is more tightly integrated with Python language and feels more native most of the times.
```
Comment (rupali317) [inline guide/english/machine-learning/pytorch-for-deep-learning/index.md]: For "In PyTorch **things** are way more imperative.....", perhaps you can briefly mention what "things" mean. Do you mean the process or something else? Otherwise it might be confusing for the user

File: guide/english/machine-learning/pytorch-for-deep-learning/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+#### What is PyTorch?
+"PyTorch is open source deep learning platform primarily developed by Facebook's artificial-intelligence research group,that provides a seamless path from research prototyping to production deployment which has been adopted by many (if not most) of the researchers that we most respect."
+
+
+* In PyTorch things are way more imperative and dynamic: you can define, change and execute nodes as you go, no special session interfaces or placeholders like in Tensorflow. Overall, the framework is more tightly integrated with Python language and feels more native most of the times.
+
+* Since computation graph in PyTorch is defined at runtime you can use our favorite Python debugging tools such as pdb, ipdb, PyCharm debugger or old trusty print statements.
+
+
+* One of the biggest features is declarative data parallelism: you can use **torch.nn.DataParallel** to wrap any module and it will be (almost magically) parallelized over batch dimension. This way you can leverage multiple GPUs with almost no effort.
```
Comment (rupali317) [inline guide/english/machine-learning/pytorch-for-deep-learning/index.md]: Let us omit "almost magically"

**Review by rupali317 — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!
